examples/charm++/ckio/iotest.ci

```diff
@@ -0,0 +1,89 @@
+mainmodule iotest {
+	include "ckio.h"; // includes the necessary functions for CkIO
+	mainchare Main {
+		entry Main(CkArgMsg* m);
+		entry void ready(Ck::IO::FileReadyMsg* msg);
+		entry void startWrite(Ck::IO::SessionReadyMsg* msg);
+		entry void postWrite(CkReductionMsg* msg);
+		entry void close(CkReductionMsg* msg);
+		entry void decrementRemaining();
+
+		entry void startWritingCycle(int file_number){
+			serial {
+				/**
+				Set the Options struct so that CkIO opens the file and sets the correct configuration for writing.
+				The writeStripe parameter determines the amount of contiguous bytes each writer chare will write to file.
+				The peStripe parameters determines how much actual data each writer chare will aggregate at a given time. This is used so that 
+				a bunch of tiny data gets distributed across many writing chares when it would be better to all go to a single chare. 
+				It is required that peStripe >= writeStripe.
+				*/
```

Comment:
Try to be a bit more concise in these descriptions of what the code is doing so the key points don't get lost too much in text.
```suggestion
				/**
				Begin the write procedure by opening the file via providing CkIO the filename and I/O options.
				*/
```